dropwizard-auth/src/main/java/io/dropwizard/auth/Demo.java

```diff
@@ -0,0 +1,34 @@
+package io.dropwizard.auth;
+
+public class Demo {
+    public static final String RPOSITORY = "testRepo";
```

Comment:
Possible typo in `RPOSITORY`, fixes:
Candidate #1 with a confidence 100%
```suggestion
    public static final String REPOSITORY = "testRepo";
```

<!-- lookout footnote separator -->
_Comment made by the analyzer style-check._